orchestrator/orchestratorAutomation.ps1

```diff
@@ -54,7 +60,7 @@ $STATIC_NODES_STRING='
     <add key="AiFabric.Logs" value="false" />
     <add key="AiFabric.ModuleEnabled" value="true" />
     <add key="AiFabric.FeatureEnabledByDefault" value="true" />
-    <add key="AiFabric.ModelStorageUrl" value="https://{{aifip}}:31443" />
+    <add key="AiFabric.ModelStorageUrl" value="https://{{aifip}}:{{storageport}}" />
     <add key="AiFabric.MLPackagingInstructionsUrl" value="something" />

```

Comment:
completely unrelated ... but can you set packaging instructionUrl value to 
https://docs.uipath.com/ai-fabric/docs/building-ml-packages

Reply:
Done